packages/file_selector/file_selector_platform_interface/lib/src/types/x_type_group/x_type_group.dart

```diff
@@ -50,6 +50,14 @@ class XTypeGroup {
         (webWildCards?.isEmpty ?? true);
   }
 
+  /// Returns the list of Uniform Type Identifiers for this group.
+  List<String>? get uniformTypeIdentifiers => macUTIs;
```

Comment:
The field should be renamed, and the aliasing be done in the other direction, because the alias will be deprecated later, not the field.

Reply:
Relatedly, the constructor needs to be updated to allow passing `uniformTypeIdentifiers`.

The behavior needs to be that we `assert` that they aren't both non-null, and then the field is set to `uniformTypeIdentifiers ?? macUTIs`.

Reply:
Hi Stuart, currently the constructor allows null values for the `macUTIs` field. Do you mean that if both `macUTIs` and `uniformTypeIdentifiers` are null, we should just throw an exception? Wouldn't it be a breaking change?

Reply:
I've been thinking about this, and now I understand that you meant if both are not null we should throw an exception, shouldn't we?

Reply:
Yes, that's correct.